src/main/java/com/ays/backend/user/model/PhoneNumber.java

```diff
@@ -0,0 +1,20 @@
+package com.ays.backend.user.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "phoneNumbers")
+@Data
```

Comment:
please make sure to check this.

Reply:
I just revised some codes regarding relationship between user and phoneNumber